examples/tls/test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+sleep 5
+/workspace/install/bin/image_client -u http://triton-server:8000 -m densenet_onnx -c 3 -s INCEPTION /workspace/images/mug.jpg
```

Comment:
Can you grep this output for the first line being classified as mug, as a sanity check?

Reply:
done